.gitlab-ci.yml

```diff
@@ -72,12 +74,27 @@ check-wasm:
         cargo check --verbose --no-default-features --target wasm32-unknown-unknown --manifest-path ${crate}/Cargo.toml;
       done
 
+build-cli:
+  stage:                           check
+  <<:                              *docker-env
+  variables:
+    GIT_STRATEGY:                    none
+  script:
+    - cargo install --git https://github.com/paritytech/cargo-contract --root ./artifacts
+    - ./artifacts/bin/cargo-contract -V
+  artifacts:
+    name:                          cli_${CI_COMMIT_SHORT_SHA}
+    expire_in:                     24 hrs
```

Comment:
So it will only be build once every 24 hours? If so this is a good start until we support it on crates.io and make it usable from `cargo-update` crate.

Reply:
Since every pipeline rebuilds it, you will have this artifact in every pipeline. 
So if it expires for the given pipeline, to restart the dependant job, you'll have to restart the builder job.